kfdefs/overlays/moc/smaug/opf-jupyterhub/blackbox-exporter.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: blackbox-exporter
+spec:
+  port:
+    targetPort: 9115-tcp
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+  to:
+    kind: Service
+    name: blackbox-exporter
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: blackbox-exporter
+    prometheus.io/scrape: "true"
+    prometheus.io/joblabel: "blackbox-metrics"
+  name: blackbox-exporter
+spec:
+  ports:
+  - name: 9115-tcp
+    port: 9115
+    protocol: TCP
+    targetPort: 9115
+  selector:
+    deployment: blackbox-exporter
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: blackbox
+data:
+  blackbox.yml: |
+    modules:
+      http_2xx:
+        prober: http
+        timeout: 10s
+        http:
+          valid_http_versions: ["HTTP/1.1", "HTTP/2.0"]
+          method: GET
+          valid_status_codes: [200, 201, 202, 203, 204, 205, 206, 207, 208, 403]
+          preferred_ip_protocol: "ip4"
+          tls_config:
+            ca_file: /etc/ssl/certs/caCertificate.crt
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    deployment: blackbox-exporter
+  name: blackbox-exporter
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      deployment: blackbox-exporter
+  template:
+    metadata:
+      labels:
+        deployment: blackbox-exporter
+    spec:
+      volumes:
+      - name: config-volume
+        configMap:
+          name: blackbox
+      - name: rh-it-ca-certificate-secret
```

Comment:
we won't need this secret/volume since this is a rh internal thing

Reply:
done